packages/components/Molecules/Card/CardEmergencyContact.vue

```diff
@@ -12,6 +12,7 @@
         :href="filterLink"
         :innerWrap="false"
         v-if="link"
+        :aria-labelledby="id"
```

Comment:
@mudrasamey rpl-link hasn't got this yet as a property, though, so it needs to be added there as well and passed on:
https://github.com/dpc-sdp/ripple/blob/develop/packages/components/Atoms/Link/Link.vue